tests/test_fixture_use.py

```diff
@@ -50,6 +50,31 @@ def test_a():  # pylint: disable=unused-variable
     suite_builder.build().run().assert_results_breakdown(success=2)
 
 
+def test_fixtures_uses_fixture(suite_builder):
+    @suite_builder.first_file.add_code
+    def __code__():  # pylint: disable=unused-variable
+        import slash
+
+        @slash.fixture
+        def fixture_1():
+            slash.context.result.data['params'] = ["fixture_1"]
+
+        @slash.fixture
+        @slash.use_fixtures(["fixture_3"])
```

Comment:
I'd use another fixture here where the order of the decorators is reversed. In the past several bugs manifested themselves this way